### ISSUE TITLE
The position and size of the set of the radio buttons in the Feedback(preference) window are not being announced by the VoiceOver

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#295]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:
1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched. 
### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio, Turn On the VoiceOver.
2. Navigate to Xamarin Inspector-> preferences->Hit On VO keys (the Control and Option keys) + space bar, which opens the General tab by default.
3. Navigate to Feedback tab-> Hit on Control+Option+Space Bar to open Feedback tab window.
4. Navigate to the below radio buttons:
a. "Yes , I am willing to participate (Recommended)" radio button.
b. "No, I would not like to participate" radio button.
5. Observe the VoiceOver announcements for these radio buttons.
### Expected Behavior
The position and size of the set of the radio buttons in the Feedback(preference) window should be announced by the VoiceOver.
Eg. For  "Yes , I am willing to participate (Recommended)" radio button VoiceOver should read as 1 of 2.
b. For "No", I would not like to participate radio button VoiceOver should read as 2 of 2.
### Actual Behavior
The position and size of the set of the radio buttons in the Feedback(preference) window are not being announced by the VoiceOver
### User Impact
The Screenreader user won't be able to know the available counts of the radio buttons.

![vo1_feedback_yesnoradiobutton_noposition sizeset](https://user-images.githubusercontent.com/22956130/39988594-63551e44-5785-11e8-8a99-5080775f8f9f.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
